packages/xo-web/src/common/selectors.js

```diff
@@ -270,6 +273,13 @@ export const isAdmin = (...args) => {
   return user && user.permission === 'admin'
 }
 
+export const getIsPoolAdmin = state =>
+  create(
+    create(createGetObjectsOfType('pool'), Object.keys),
+    getCheckPermissions,
+    (poolsIds, check) => some(poolsIds, poolId => check(poolId, 'administrate'))
+  )
```

Comment:
There's no need for it to be a function.
```js
export const getIsPoolAdmin = create(...)
```
Then don't forget to pass the selector directly whenever you use it:
```js
isPoolAdmin: getIsPoolAdmin,
```